src/zabapgit_objects.prog.abap

```diff
@@ -2,6 +2,26 @@
 *&  Include           ZABAPGIT_OBJECTS
 *&---------------------------------------------------------------------*
 
+* Macros
+
+DEFINE object_check_timestamp.
+  IF sy-subrc = 0 AND &1 IS NOT INITIAL AND &2 IS NOT INITIAL.
+    cl_abap_tstmp=>systemtstmp_syst2utc(
+      EXPORTING syst_date = &1
+                syst_time = &2
+      IMPORTING utc_tstmp = lv_ts ).
+    IF lv_ts < iv_timestamp.
+      rv_changed = abap_false. " Unchanged
+    ELSE.
+      rv_changed = abap_true.
+      RETURN.
```

Comment:
can we replace the macro with a method? It is not very typical to have RETURN in a macro definition?